applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -151,6 +151,28 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ElasticIsotropic3D
         return StressMeasure_Cauchy;
     }
 
+    /**
+     * @brief Adds the initial stress vector if it is defined in the InitialState
+     */
+    void AddInitialStressVectorContribution(Vector& rStressVector, Parameters& rParameterValues) 
+    {
+        if (this->HasInitialState()) {
+            const auto& r_initial_state = GetInitialState();
+            noalias(rStressVector) += r_initial_state.GetInitialStressVector();
+        }
+    }
+
+    /**
+     * @brief Adds the initial strain vector if it is defined in the InitialState
+     */
+    void AddInitialStrainVectorContribution(Vector& rStrainVector, Parameters& rParameterValues)
+    {
+        if (this->HasInitialState()) {
+            const auto& r_initial_state = GetInitialState();
+            noalias(rStrainVector) += r_initial_state.GetInitialStrainVector();
+        }
+    }
```

Comment:
ping can you please try again to move this to the constitutive law?
Otherwise this will be duplicated in every law

Reply:
I can do it! @RiccardoRossi opinion?